projects/igniteui-angular/src/lib/select/select.component.html

```diff
@@ -40,7 +40,8 @@
     </div>
 
     <!-- #7436 LMB scrolling closes items container - unselectable attribute is IE specific  -->
-    <div #scrollContainer class="igx-drop-down__list-scroll" unselectable="on" [style.maxHeight]="maxHeight" [attr.id]="this.listId" role="listbox">
+    <div #scrollContainer class="igx-drop-down__list-scroll" unselectable="on" [style.maxHeight]="maxHeight"
+        [attr.id]="this.listId" role="listbox" [attr.aria-labelledby]="this.label ? this.label.id : ''">
```

Comment:
This way will be shorter :)
```suggestion
        [attr.id]="this.listId" role="listbox" [attr.aria-labelledby]="this.label?.id">
```